Tools/fix_code_style.sh

```diff
@@ -1,4 +1,13 @@
-#!/bin/sh
+#!/bin/bash
+
+ASTYLE_VER=`astyle --version`
+ASTYLE_VER_REQUIRED="Artistic Style Version 2.05.1"
```

Comment:
I think if we want to be so picky (and I agree that we have to control the version) then we would need to build it from source on each platform off a git repo. Else we need some sort of soft matching.